controllers/user.controller.js

```diff
@@ -22,101 +21,93 @@ exports.create = (req, res) => {
-        message:
-          err.message || "Some error occurred while while looking for users.",
-      });
+exports.findAll = async (req, res) => {
+  try {
+    const users = await User.findAll();
+    res.send(users);
+  } catch (err) {
+    res.status(500).send({
+      message: err.message || "Some error occurred while looking for users.",
     });
+  }
 };
 
-exports.findOne = (req, res) => {
-  const id = req.params.id;
-  User.findByPk(id)
-    .then((data) => {
-      if (data) res.send(data);
-      else {
-        res.status(404).send({
-          message: `Cannot find user with id: ${id}`,
-        });
-      }
-    })
-    .catch((err) => {
-      res.status(500).send({
-        message: err.message || "Some error occurred while looking for user.",
+exports.findOne = async (req, res) => {
+  try {
+    const id = req.params.id;
+    const user = await User.findByPk(id);
+    if (user) return res.send(user);
+    else {
+      return res.status(404).send({
+        message: `Cannot find user with id: ${id}`,
       });
+    }
+  } catch (err) {
+    return res.status(500).send({
+      message: err.message || "Some error occurred while looking for user.",
     });
+  }
 };
 
-exports.update = (req, res) => {
-  const id = req.params.id;
-  User.update(req.body, {
-    where: { id: id },
-  })
-    .then((num) => {
-      if (num == 1) {
-        res.send({
-          message: "User was updated successfully.",
-        });
-      } else {
-        res.send({
-          message: `Cannot update User with id=${id}. Maybe User was not found or req.body is empty!`,
-        });
-      }
-    })
-    .catch((err) => {
-      res.status(500).send({
-        message: "Error updating User with id=" + id,
+exports.update = async (req, res) => {
+  try {
+    const id = req.params.id;
+    const response = await User.update(req.body, {
+      where: { id: id },
+    });
+    if (response == 1) {
+      return res.send({
+        message: "User was updated successfully.",
+      });
+    } else {
+      return res.send({
+        message: `Cannot update User with id=${id}. Maybe User was not found or req.body is empty!`,
       });
+    }
+  } catch (err) {
+    return res.status(500).send({
+      message: "Error updating User with id=" + id,
     });
+  }
 };
 
-exports.destroy = (req, res) => {
-  const id = req.params.id;
-  User.destroy({
-    where: { id: id },
-  })
-    .then((num) => {
-      if (num == 1) {
-        res.send({
-          message: "User was deleted successfully!",
-        });
-      } else {
-        res.send({
-          message: `Cannot delete User with id=${id}. Maybe User was not found!`,
-        });
-      }
-    })
-    .catch((err) => {
-      res.status(500).send({
-        message: "Could not delete User with id=" + id,
+exports.destroy = async (req, res) => {
+  try {
+    const id = req.params.id;
+    const response = await User.destroy({
+      where: { id: id },
+    });
+    if (response == 1) {
```

Comment:
=== instead of == - read about difference

Reply:
respons in my case is an array with one value, thats why I chose the '==' sign. I can change it to `response[0] === 1`, but I will try to find a better way.

Reply:
seems like `if(response)` works just fine :sweat_smile: